tests/scripts/test_detached_actor_2.py

```diff
@@ -0,0 +1,27 @@
+import ray
+import time
+import sys
+
+ray.init(address='ray://127.0.0.1:10001', namespace=sys.argv[1])
+
+def retry_with_timeout(func, timeout=90):
+    err = None
+    for _ in range(timeout):
+        try:
+            return func()
+        except BaseException as e:
```

Comment:
Back to Exception

Reply:
We do not need to back to Exception. `timeout` will not be caught by BaseException (in other words, the process can still be killed by the `timeout` command). 

The difference between BaseException and Exception is:
```bash
python3 tests/scripts/test_detached_actor_2.py [NAMESPACE]
Ctrl + C => The process will not be terminated (caught by BaseException).

# The process will be killed after 5 seconds.
timeout 5s python3 tests/scripts/test_detached_actor_2.py [NAMESPACE] 
```

cc @DmitriGekhtman

Reply:
sgtm